### ISSUE TITLE
Regarding line 58 in preprocess .py

### ISSUE THREAD

**Issue by saba56** [Kyfafyd/MirrorGAN#10]
Could you please let me know what is the purpose of line
 for c in filelist2:
        if c == 'Muscular' or c == 'Circulatory' or c == 'Dental_Tooth':
            continue
        ff = os.listdir('data/health/CUB_200_2011/images/' + c)

**Comment by Kyfafyd** [comment]
I have tried to apply MirrorGAN to a medical image generation task. So this is specified for that task, you can generate for your own by modifying this.

**Comment by saba56** [comment]
Thanks for the reply....But when I generate the class_info.pickle and the filenames .pickle for birds dataset using this code, the files do not resemble the original filenemes.pickle and class-info.pickle that are there in the Birds dataset.....More importantly class_info.pickle has entire different format.....

**Comment by Kyfafyd** [comment]
That is just a reference, I think this part is not difficult, you can further try it on your own.

**Comment by saba56** [comment]
Yes but i am not able to make out the things from class_info.pickle....What do below parameters indicate
 
(lp0
cnumpy.core.multiarray
scalar
p1
(cnumpy
dtype
p2
(S'i8'
p3
I0
I1
tp4
Rp5
(I3
S'<'
p6
NNNI-1
I-1
I0
tp7
bS'\x02\x00\x00\x00\x00\x00\x00\x00'
p8
tp9
